src/amo/reducers/users.js

```diff
@@ -449,6 +495,20 @@ const reducer = (
         ...state,
         currentUserID: null,
       };
+    case UNLOAD_USER_ACCOUNT: {
+      const { userId } = action.payload;
+
+      const newState = { ...state };
+
+      if (newState.byID[userId]) {
+        const { username } = newState.byID[userId];
+
+        delete newState.byID[userId];
+        delete newState.byUsername[username];
```

Comment:
Should `currentUserID` also be set to `null` in here, or is that done elsewhere, prior to calling `UNLOAD_USER_ACCOUNT`?

Reply:
interesting that you noticed this here, I completely forgot when writing this patch. it is done in the next patch though, can we keep things as is here or do you want me to update the code here?

Reply:
I think it's fine to land this as is and accept the change in the next patch (which I noticed). 😄